buildenv/jenkins/variables/defaults.yml

```diff
@@ -271,7 +271,10 @@ ppc64_aix:
     8: 'aix-ppc64-normal-server-release'
     11: 'aix-ppc64-normal-server-release'
   node_labels:
-    build: 'ci.role.build && hw.arch.ppc64 && sw.os.aix.7_2'
+    build:
+      all: 'ci.role.build && hw.arch.ppc64 && sw.os.aix.7_2 && !sw.tool.c++runtime.17_1'
```

Comment:
Like https://github.com/eclipse-openj9/openj9/pull/22788, pls require `sw.tool.c++runtime.16_1`

Reply:
> I think I'd much prefer sw.tool.c++runtime.16_1 over the !sw.tool.c++runtime.17_1, but I am not sure that is the runtime installed or even labeled on all the test systems. Would be easy to update the node configs though.

The test systems don't matter, only the build systems, which are already updated.